src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+class LottoNumber private constructor(
+    val value: Int,
+) : Comparable<LottoNumber> {
+    override fun compareTo(other: LottoNumber): Int = value.compareTo(other.value)
```

Comment:
compareTo는 infix함수이기에 dot(.) 없이도 표현이 가능하죠 ㅎㅎ

```suggestion
    override fun compareTo(other: LottoNumber): Int = value compareTo other.value
```